skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/db/DatabaseSchema.kt

```diff
@@ -104,7 +99,6 @@ class Brevredigering(id: EntityID<Long>) : LongEntity(id) {
     var spraak by BrevredigeringTable.spraak
     var avsenderEnhetId by BrevredigeringTable.avsenderEnhetId
     var saksbehandlerValg by BrevredigeringTable.saksbehandlerValg
-    private var _redigertBrev by BrevredigeringTable.redigertBrev.writeHashTo(BrevredigeringTable.redigertBrevHash)
     private var redigertBrevKryptert by BrevredigeringTable.redigertBrevKryptert.writeHashTo(BrevredigeringTable.redigertBrevKryptertHash)
```

Comment:
Om du fjerner `Kryptert` fra navnet på disse to variablene nå, så trenger vi ikke get/set lenger.